injector.js

```diff
@@ -36,15 +52,22 @@ const injectScriptList = scriptList => {
 
         injectSettings(settings); // Give the DOM access to settings
 
+        // Disable scripts from loading if they are not needed
         if (!settings.calculation && !settings.quickview)
-            scriptList.calculation.enabled = false; // Disable the script from loading
+            scriptList.calculation.enabled = false; 
 
         if (!settings.liveModification)
-            scriptList.liveModification.enabled = false; // Disable the script from loading
+            scriptList.liveModification.enabled = false; 
         
-        if (!settings.percentageColumn)
-            scriptList.percentageColumn.enabled = false; // Disable the script from loading
+        if (!settings.percentages)
+            scriptList.percentages.enabled = false; 
 
+        if (!settings.betterTableLayout)
+            scriptList.betterTableLayout.enabled = false;
+        
+        if (!settings.percentages && !settings.liveModification)
```

Comment:
```suggestion
        if (!settings.percentages && !settings.liveModification && !settings.betterTableLayout)
```
You still need to load the selectors if you are going to use the table layout

Reply:
The better table layout feature runs before any other modifications to the table layout, so the jQuery selectors are static. As such, the selectors do not need to be loaded if the better table layout is the only enabled feature.

Reply:
Try it out, only enable the better table layout without having anything else enabled. You will get an error.

Reply:
Thanks for pointing it out. Although the issue is within loadMarkbook.js, as getSelectors() is always called, but that will be fixed.